docs/ext/ipc.md

```diff
@@ -0,0 +1,200 @@
+* `unavailable`: There were no servers available to process the request.
+* `throttled`: The request was rejected due to the client or server considering the server to
+  be above capacity.
+* `timeout`: The request could not or would not be complete within the configured threshold
+  (either on client or server).
+* `cancelled`: The client cancelled the request before it was completed.
+* `access_denied`: The request was denied access for authentication or authorization reasons.
+
+## Server Metrics
+
+### ipc.server.call
+
+This is a [percentile timer] that is recorded for each inbound message to a server.
+
+**Dimensions:**
+
+* `ipc.protocol`
+* `ipc.result`
+* `ipc.vip`
+* `ipc.endpoint`
+* `ipc.status`
+* `ipc.status.detail`
+* `ipc.failure.injected`
+* `ipc.attempt`
+* `ipc.client.app`
+* `ipc.client.asg`
+* `owner`
+* `id`
+
+### ipc.server.call.size.inbound
+
+This is a [distribution summary] of the size in bytes of inbound messages received by a server.
+
+**Dimensions:**
+
+* `ipc.protocol`
+* `ipc.vip`
+* `ipc.endpoint`
+* `ipc.result`
+* `ipc.status`
+* `ipc.status.detail`
+* `ipc.client.app`
+* `ipc.client.asg`
+* `owner`
+* `id`
+
+### ipc.server.call.size.outbound
+
+This is a [distribution summary] of the size in bytes of outbound messages sent from a server.
+
+**Dimensions:**
+
+* `ipc.protocol`
+* `ipc.vip`
+* `ipc.endpoint`
+* `ipc.result`
+* `ipc.status`
+* `ipc.status.detail`
+* `ipc.client.app`
+* `ipc.client.asg`
+* `owner`
+* `id`
+
+### ipc.server.inflight
+
+This is a [distribution summary] that shows the number of inbound IPC messages currently being
+processed in a server.
+
+**Dimensions:**
+
+* `ipc.protocol`
+* `ipc.endpoint`
+* `ipc.client.app`
+* `ipc.client.asg`
+* `owner`
+* `id`
+
+## Client Metrics
+
+### ipc.client.call
+
+This is a [percentile timer] that is recorded for each outbound message from a client.
+
+**Dimensions:**
+
+* `ipc.protocol`
+* `ipc.result`
+* `ipc.vip`
+* `ipc.endpoint`
+* `ipc.status`
+* `ipc.status.detail`
+* `ipc.failure.injected`
+* `ipc.attempt`
+* `ipc.attempt.final`
+* `ipc.server.app`
+* `ipc.server.asg`
+* `owner`
+* `id`
+
+### ipc.client.call.size.inbound
+
+This is a [distribution summary] of the size in bytes of inbound messages received by a client.
+
+**Dimensions:**
+
+* `ipc.protocol`
+* `ipc.vip`
+* `ipc.endpoint`
+* `ipc.server.app`
+* `ipc.server.asg`
+* `owner`
+* `id`
+
+### ipc.client.call.size.outbound
+
+This is a [distribution summary] of the size in bytes of outbound messages sent from a client.
+
+**Dimensions:**
+
+* `ipc.protocol`
+* `ipc.vip`
+* `ipc.endpoint`
+* `ipc.result`
+* `ipc.status`
+* `ipc.status.detail`
+* `ipc.server.app`
+* `ipc.server.asg`
+* `owner`
+* `id`
+
+### ipc.client.inflight
+
+This is a [distribution summary] that shows the number of currently outstanding outbound
+IPC messages from a client.
+
+**Dimensions:**
+
+* `ipc.protocol`
+* `ipc.vip`
+* `ipc.endpoint`
+* `ipc.result`
+* `ipc.status`
+* `ipc.status.detail`
```

Comment:
Status/result should be remove here correct? Since the inflight metric needs to get updated before the result is received, I don't think it is possible to include that information on the inflight tracker.

Reply:
Yes, my mistake.